ROMFS/px4fmu_common/init.d/rc.fw_defaults

```diff
@@ -1,11 +1,16 @@
 #!nsh
+#
+# Fixed wing default parameters
+#
+# NOTE: Script variables are declared/initialized/unset in the rcS script.
+#
 
 set VEHICLE_TYPE fw
 
 if [ $AUTOCNF == yes ]
 then
 	#
-	# Default parameters for FW
+	# Default parameters for fixed wing uavs.
```

Comment:
UAV is an acronym

Reply:
Haha!  Hamish! You get where I am coming from. :)  Let me know your preference!

Reply:
I'm OK with this, but if you do any other tidy, suggest you do `# Default parameters for fixed wing UAVs.`

Reply:
Excellent! I understand! Thank you!